tests/test_funcs.py

```diff
@@ -1271,6 +1270,10 @@ def test_shapes(
     ):
         x1 = xp.zeros(shape1)
         x2 = xp.zeros(shape2)
+
+        if is_jax_namespace(xp) and assume_unique and shape1 != (1,):
+            pytest.xfail(reason="jax#32335 fixed with jax>=0.8.0")
```

Comment:
This terminates the test early. I would like this test to XPASS (which is a failure) when we upgrade the pixi.lock, to force the maintainer to clean this up.
Instead of `pytest.xfail`, please use https://github.com/data-apis/array-api-extra/blob/ebe9a5be9826ee4c82f8aaf56c19ffa36b3a005d/src/array_api_extra/_lib/_testing.py#L279